src/main/java/Duke.java

```diff
@@ -7,8 +7,7 @@ public static void main(String[] args) {
         String userInput = "";
         int count = 0;
 
-        System.out.println("Hey! I am " + logo);
-        System.out.println("What would you like to do?");
+        displayWelcome(logo);
```

Comment:
These three codes seem that they can be all wrapped into "displayWelcome(logo)"

Reply:
Thats a good idea, I will implement it into my code :)